### PR TITLE
[CORE24-358] fix(individuals): Correct ordering for fields in form

### PR DESCRIPTION
**Pull request by dfigueroa87**
Jira ticket: JIRA-CORE24-358

## What have you implemented?

Re-ordered the individual fields according to the updated spreadsheet.

## How to test it?

Go to create/edit/view a individual, check the order and the sections match to what's specified in the Global Registration Form spreadsheet.

## Extra/next steps

There's two missing fields:
- Collect agent title
- Upload agent

That will need to be added once we include roles to the system.

### CODE REVIEW

File: libs/prm-components/src/lib/config/participant.ts [hunk 1]
```diff
@@ -23,26 +23,30 @@ export const participantConfig: EntityUIConfigLoader = (staticData) => ({
...
             label: 'Consent GDPR',
-            required: true,
           },
           {
             path: ['consentReferral'],
             dataType: DataType.Boolean,
             component: Component.Checkbox,
             label: 'Consent Referral',
-            required: true,
           },
```
Comment (internetti) [inline libs/prm-components/src/lib/config/participant.ts:32]: according to fix's table these two are supposed to be in the protcetion section?

**Review by internetti — COMMENTED** [review]
kinda hard to tell from the new grf form which fields should already be there, what about the free fields for example, do we add them later?

**Comment by dfigueroa87** [comment]
> kinda hard to tell from the new grf form which fields should already be there, what about the free fields for example, do we add them later?

I think you may be looking at a different spreadsheet. This is the one I got from Fix and used: https://norwegianrefugeecouncil.sharepoint.com/:x:/r/sites/HO-D-Team-Experiments-COREProduct/_layouts/15/Doc.aspx?sourcedoc=%7B0C17E00D-636B-41B8-A6F6-77B8512903BD%7D&file=240605%20-%20Global%20Registration%20Form%20-%20V2-3.xlsx&wdOrigin=TEAMS-WEB.p2p_ns.rwc&action=default&mobileredirect=true

File: libs/prm-components/src/lib/config/participant.ts (hunk 1 above)
Comment (neb42) [reply libs/prm-components/src/lib/config/participant.ts:32]: Worth running this by him as I think they're much better here

Comment (dfigueroa87) [reply libs/prm-components/src/lib/config/participant.ts:32]: The version of the spreadsheet I'm looking at has them in the Consent section

**Comment by internetti** [comment]
> > kinda hard to tell from the new grf form which fields should already be there, what about the free fields for example, do we add them later?
> 
> I think you may be looking at a different spreadsheet. This is the one I got from Fix and used: https://norwegianrefugeecouncil.sharepoint.com/:x:/r/sites/HO-D-Team-Experiments-COREProduct/_layouts/15/Doc.aspx?sourcedoc=%7B0C17E00D-636B-41B8-A6F6-77B8512903BD%7D&file=240605%20-%20Global%20Registration%20Form%20-%20V2-3.xlsx&wdOrigin=TEAMS-WEB.p2p_ns.rwc&action=default&mobileredirect=true

ah I was looking at V1 ...

**Review by internetti — APPROVED** [review]

**Pull request merged**